app/views/oidc/api_key_roles/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% @title = t(".api_key_roles") %>
+
+<div class="t-body">
+  <table class="owners__table">
+    <thead>
+      <tr class="owners__row owners__header">
+        <th class="owners__cell">
+          <%= OIDC::ApiKeyRole.human_attribute_name(:name) %>
+        </th>
+        <th class="owners__cell">
+          <%= OIDC::ApiKeyRole.human_attribute_name(:token) %>
+        </th>
+        <th class="owners__cell">
+          <%= OIDC::Provider.human_attribute_name(:issuer) %>
+        </th>
+      </tr>
+    </thead>
+    <tbody class="t-body">
+      <% @api_key_roles.each do |api_key_role| %>
+        <tr class="owners__row">
+          <td class="owners__cell" data-title="Name">
+            <%= link_to api_key_role.name, profile_oidc_api_key_role_path(api_key_role.token) %>
```

Comment:
It's a bit hard to tell that the link goes to the show page. Can we add a `Details` button on the side of each row like the edit button in API keys?
<img width="547" alt="Screenshot 2023-08-14 at 8 25 11 PM" src="https://github.com/rubygems/rubygems.org/assets/42748004/5cee3588-a1b9-4215-a526-6e0665c38bba">